bounded/src/Data/Bounded/Integral.hs

```diff
@@ -46,6 +46,14 @@ instance
       minBound' :: Natural
       minBound' = convert @Natural $ minBound @bound
 
+instance
+  ( HasValidTypeRange '(min2, min1) (min1 <=? min2)
```

Comment:
__question:__ Is there a `>=?` operator so that the order of the operands can remain consistent? I think it's easier to read/understand if it uses the same operand order, but different operator.

Reply:
No, unless I import it in some other package or create it myself. It was my first pick as well thinking it exists.

Reply:
@epicallan the `~` operator says they are the same _type_ of thing, not necessarily the *exact* same value, correct?

Reply:
`Data.Type.Ord` is only in 4.16 which is in GHC-9.2 and not GH-9.0 so going to leave it as is for now.